src/en/RainOfSnow.lua

```diff
@@ -15,10 +32,10 @@ local function expandURL(url)
 end
 
 local function getPassage(chapterURL)
-	local chap = GETDocument(chapterURL):selectFirst("div.content")
-	-- Removes the title and makes it an H1 for consistant custom CSS.
-	local title = chap:selectFirst("h2"):text()
-	chap:selectFirst("h2"):remove()
+	-- The additional shrinkURL is to support older version of the extension. (That didn't have shortened links)
+	local content = GETDocument(expandURL(shrinkURL(chapterURL)))
+	local title = content:selectFirst("li.menu-toc-current"):text()
+	local chap = content:selectFirst(".zoomdesc-cont")
 	chap:child(0):before("<h1>" .. title .. "</h1>")
```

Comment:
Might as well use content instead of a new variable as content does not get used anymore after this.

Reply:
Changed to be so.